Count Macro - added option to supply a list

danjhd opened the pull request:
*Issue #, if available:* n/a

*Description of changes:*
Enhancement to the Count Macro. Added capability to supply a `List` in addition to the existing functionality of using an `Integer` value.
Existing functionality remains unchanged.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

nathanpeck commented:
This is a very cool PR, however I'd like for it to be additive rather than removing the existing decimal example. Could you perhaps add the new macro as a new section in the text file, and a new template, or new section in the existing macro example template?

danjhd commented:
> This is a very cool PR, however I'd like for it to be additive rather than removing the existing decimal example. Could you perhaps add the new macro as a new section in the text file, and a new template, or new section in the existing macro example template?

@nathanpeck i apologise i was not 100% on what you were referring to. I have assumed it was the part of the documentation that shows the placeholder replacement? I had removed the example that uses `%d` alone.
I have now updated the example to put that example back in and add the new example as a second resource.
Is this what you were looking for?

nathanpeck commented:
Gotcha! Looks good!
